src/main/java/com/sap/sgs/phosphor/fosstars/model/value/PackageManagers.java

```diff
@@ -0,0 +1,110 @@
+package com.sap.sgs.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * A set of package managers.
+ */
+public class PackageManagers {
```

Comment:
Why do we need this class?  Set<PackageManager> or List<PackageManager> would be the same right?

Reply:
Right, that's just a set. I am even thinking about implementing a set interface. I would prefer to keep this class (and other similar ones such as `Languages`). The class allows adding convenient methods if necessary, and also using `Value<PackageManagers` instead of `Value<Set<PackageManager>>` which is not a big deal but looks nicer to me.